ZEN.md

```diff
@@ -0,0 +1,66 @@
+# Mobile RUM SDK Philosophy
+
+## TL;DR;
```

Comment:
I am not sure this format is correct, if you look at the file rich view mode it looks broken